dashboard/modules/job/cli.py

```diff
@@ -314,4 +315,31 @@ def logs(address: Optional[str], job_id: str, follow: bool):
                 "for this feature."
             )
     else:
-        print(client.get_job_logs(job_id), end="")
+        # Set no_format to True because the logs may have unescaped "{" and "}"
+        # and the CLILogger calls str.format().
+        cli_logger.print(client.get_job_logs(job_id), end="", _no_format=True)
```

Comment:
nit, but if we're using this `_no_format` flag, it should not be `_` prefixed

Reply:
Changing to cli_logger here doesn't affect the format right?

Reply:
Or STDOUT vs. STDERR ?

Reply:
Should we edit the CLILogger API to change `_no_format` to `no_format`, or skip using the CLILogger altogether?

I'll double check the STDOUT, STDERRand the formatting.

Reply:
change it to `no_format`

Reply:
Done.  Tested STDOUT and STDERR before and after the change; they were the same.

STDOUT: (colors don't work here but they show up fine in the terminal)
```
[37mJob submission server address[39m: [1mhttp://127.0.0.1:8265[22m

[32m-------------------------------------------------------[39m
[32mJob 'raysubmit_LAQnLBKDs1LuUHEF' submitted successfully[39m
[32m-------------------------------------------------------[39m

[36mNext steps[39m
  Query the logs of the job:
    [1mray job logs raysubmit_LAQnLBKDs1LuUHEF[22m
  Query the status of the job:
    [1mray job status raysubmit_LAQnLBKDs1LuUHEF[22m
  Request the job to be stopped:
    [1mray job stop raysubmit_LAQnLBKDs1LuUHEF[22m

Tailing logs until the job exits (disable with --no-wait):
1
2
3
4
5
6
7
8
9
10
2.26.0

[32m------------------------------------------[39m
[32mJob 'raysubmit_LAQnLBKDs1LuUHEF' succeeded[39m
[32m------------------------------------------[39m

```

STDERR:
```
2022-03-01 15:15:41,728	INFO sdk.py:229 -- Uploading package gcs://_ray_pkg_3dd0f75076634f30.zip.
2022-03-01 15:15:41,729	INFO packaging.py:367 -- Creating a file package for local directory './'.
```